src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const arrList = document.querySelectorAll('li');
+const arrDate = [];
+
+function sortList(list) {
+  for (let i = 0; i < arrList.length; i++) {
+    arrDate.push({
+      name: arrList[i].textContent,
+      position: arrList[i].dataset.position,
+      salary: arrList[i].dataset.salary,
+      age: arrList[i].dataset.age,
+    });
+  }
+
+  return arrDate;
+}
+
+function getEmployees(list) {
+  arrDate.sort(
+    (el1, el2) =>
+      +el2.salary.slice(1).split(',').join('') -
+      +el1.salary.slice(1).split(',').join(''),
```

Comment:
The salary conversion logic assumes that the salary is always in a specific format. Consider adding error handling or checks to ensure the format is correct before attempting to convert it.